### PR TITLE
GiveNamedItem using virtual function

### PR DESCRIPTION
**Pull request by komashchenko**
Replaced GiveNamedItem call with a virtual function

### CODE REVIEW

File: addresses.h [hunk 1]
```diff
@@ -50,7 +49,6 @@ namespace sigs
 	inline const byte *StateChanged = (byte *)"\x48\x89\x54\x24\x10\x55\x53\x57\x41\x55";
 	inline const byte *GiveNamedItem = (byte *)"\x48\x89\x5C\x24\x18\x48\x89\x74\x24\x20\x55\x57\x41\x54\x41\x56\x41\x57\x48\x8D\x6C\x24\xD9";
 
-	inline const byte *VoiceShouldHear = (byte *)"\x48\x89\x5C\x24\x10\x44\x88\x4C\x24\x20\x44\x88\x44\x24\x18";
```
Comment (xen-000) [inline addresses.h:53]: Was this removed by mistake?

File: addresses.h [hunk 2]
```diff
@@ -78,7 +76,6 @@ namespace sigs
 	inline const byte *StateChanged = (byte *)"\x55\x48\x89\xE5\x41\x57\x41\x56\x41\x55\x41\x54\x53\x89\xD3";
 	inline const byte *GiveNamedItem = (byte *)"\x55\x48\x89\xE5\x41\x57\x41\x56\x49\x89\xCE\x41\x55\x49\x89\xF5\x41\x54\x49\x89\xD4";
 
-	inline const byte *VoiceShouldHear = (byte *)"\x55\x48\x89\xE5\x41\x57\x41\x89\xD7\x41\x56\x49\x89\xF6\x41\x55\x41\x54\x49\x89\xFC\x53\x48\x83\xEC\x28";
```
Comment (xen-000) [inline addresses.h:81]: Ditto

**Comment by xen-000** [comment]
I assume it's caused by the conflict, either way this works so nice job 👍

**Comment by komashchenko** [comment]
Fixed, apparently I deleted the wrong signature

**Pull request merged**